code/controllers/configuration/entries/jobs.dm

```diff
@@ -133,6 +133,7 @@
 		var/list/working_list = list()
 		for(var/config_datum_key in job_config_datum_singletons)
 			var/datum/job_config_type/config_datum = job_config_datum_singletons[config_datum_key]
+			if(!config_datum.validate_entry(occupation)) continue // Dont make the entry if it doesn't apply to this job
```

Comment:
use a newline do not one line anything it makes things harder to read. goes for the other two instances too